### PR TITLE
Add redirects for renamed files

### PR DESCRIPTION
**Pull request by lornajane**
# What changed, and why it matters

When we move or rename a file (which should be avoided if possible), it breaks the URL since the URL is made from the directory and file names. We do have a redirects setup to handle the special cases where we have moved something, however we have had some broken links and redirects reported when our SEO agency were doing some analysis on the docs. This pull request updates the reported URLs.

### CODE REVIEW

File: _redirects [hunk 1]
```diff
@@ -39,3 +39,6 @@ https://developer.aiven.io/* https://docs.aiven.io/:splat 301!
 # Renamed/deleted files
 /docs/products/flink/howto/real-time-alerting-solution-cli.html       /docs/products/flink/howto/real-time-alerting-solution.html
 /docs/platform/howto/disable-user-2fa.html                            /docs/platform/howto/user-2fa.html
+/docs/products/mysql/howto/max-number-of-connections.html             /docs/products/mysql/concepts/max-number-of-connections.html
+/docs/products/postgresql/reference/plan-iops.html                    /docs/products/postgresql/reference/resource-capability.html
+/docs/tools/terraform/upgrade-provider-v1-v2                          /docs/tools/terraform/howto/upgrade-provider-v1-v2
```
Comment (angelinekwan) [inline _redirects:44]: This redirect doesn't seems to work - notice in [preview](https://deploy-preview-1322--developer-aiven-io-preview.netlify.app/docs/tools/terraform/upgrade-provider-v1-v2). Will test something out in this PR

File: _redirects [hunk 2]
```diff
@@ -38,4 +38,7 @@ https://developer.aiven.io/* https://docs.aiven.io/:splat 301!
 
 # Renamed/deleted files
 /docs/products/flink/howto/real-time-alerting-solution-cli.html       /docs/products/flink/howto/real-time-alerting-solution.html
-/docs/platform/howto/disable-user-2fa.html                            /docs/platform/howto/user-2fa.html
+/docs/platform/howto/disable-user-2fa.html                            /docs/platform/howto/user-2fa.html                                  301!
```
Comment (angelinekwan) [inline _redirects:41]: Tried adding the force redirect but didn't resolve the redirect loop - can see in this [PR preview](https://deploy-preview-1322--developer-aiven-io-preview.netlify.app/docs/platform/howto/disable-user-2fa.html). Will open an Netlify support ticket. Similar case for `/docs/tools/terraform/upgrade-provider-v1-v2.html`

Comment (angelinekwan) [reply _redirects]: @lornajane Netlify Support suggested to swap the order of the redirect rules above 
`# Redirect from .index.html to specific page names for landing`

And now the following 2 redirect loop are resolved and redirecting to the correct url.
https://deploy-preview-1322--developer-aiven-io-preview.netlify.app/docs/platform/howto/disable-user-2fa.html
https://deploy-preview-1322--developer-aiven-io-preview.netlify.app/docs/tools/terraform/upgrade-provider-v1-v2.html

**Review by angelinekwan — APPROVED** [review]

File: _redirects (hunk 2 above)
Comment (lornajane) [reply _redirects]: Oh that sort of makes sense now. Thank you for fixing this!

**Pull request merged**